packages/create-next-app/index.ts

```diff
@@ -136,13 +143,52 @@ async function run(): Promise<void> {
   }
 
   const example = typeof program.example === 'string' && program.example.trim()
+
+  /**
+   * If the user does not provide the necessary flags, prompt them for whether
+   * to use TS or JS.
+   *
+   * @todo Allow appDir to support TS or JS, currently TS-only and disables all
+   * --ts, --js features.
+   */
+  if (!program.experimentalApp && !program.typescript && !program.javascript) {
+    const styledTypeScript = chalk.hex('#007acc')('TypeScript')
+    const { typescript } = await prompts(
+      {
+        type: 'toggle',
+        name: 'typescript',
+        message: `Would you like to use ${styledTypeScript} with this project?`,
+        initial: true,
+        active: 'Yes',
+        inactive: 'No',
+      },
+      {
+        /**
+         * User inputs Ctrl+C or Ctrl+D to exit the prompt. We should close the
+         * process and not write to the file system.
+         */
+        onCancel: () => {
+          console.error('Exiting.')
+          process.exit(1)
+        },
+      }
+    )
+
+    if (typescript) {
+      program.typescript = true
+    } else {
+      program.javascript = false
```

Comment:
```suggestion
      program.typescript = false
```

Reply:
Not quite the right change, but thank you for catching that redundant first assignment. I edited to make it clear that the program variables were being assigned depending on the prompt response.

In a future refactor, it would be ideal to simply set a `template: "js" | "ts" | "app"` variable rather than alternating booleans, sort of hacking an enum in with `if/else` statements.